requirements.txt

```diff
@@ -7,4 +7,5 @@ coverage==4.1
 nose==1.3.7
 tornado==4.0.2
 flake8==2.5.2
+typing==3.6.2
```

Comment:
Does the `typing` library at all conflict with Python `typing` module (>=3.5)?

Reply:
I think it doesn't, in that the stdlib one will be imported instead of this one.